tree/ntuple/v7/src/RColumn.cxx

```diff
@@ -44,22 +44,15 @@ ROOT::Experimental::Internal::RColumn::~RColumn()
 void ROOT::Experimental::Internal::RColumn::ConnectPageSink(DescriptorId_t fieldId, RPageSink &pageSink,
                                                             NTupleSize_t firstElementIndex)
 {
-   fPageSink = &pageSink; // the page sink initializes fWritePage on AddColumn
+   if (pageSink.GetWriteOptions().GetInitialNElementsPerPage() * fElement->GetSize() >
+       pageSink.GetWriteOptions().GetMaxUnzippedPageSize()) {
+      throw RException(R__FAIL("initial number of elements per page too small"));
```

Comment:
hm, isn't it the other way around? The maximum unzipped page size is too small for the initial number of elements...